### DISCUSSION TITLE
整理Issues

### DISCUSSION THREAD

**Discussion by wbccb** [didi/LogicFlow#1186] [General]
目前`Issues`数量过多，有108个，很小部分是`bug`，绝大部分都是`优化/新功能`开发，有没有计划整理一下，关闭一些不积极的`Issues`，将一些认为可以做的`优化/新功能`列出来纳入版本规划

现在太多`new feature`和`优化`=_=我想帮忙做几个，都不知道该选哪个入手？

**Comment by towersxu** [comment]
@wbccb 非常欢迎一起参与进来！您可以优先从提出时间比较近的issue入手，时间比较久issue的大概率是功能影响不大或者有其他替代方案。对于issue的整理，我们非常欢迎您一起参与进来。若您对LogicFlow开发有兴趣，您可以添加微信号`logic-flow`，备注上你的github名称，我们研发同学会添加您好友，同你一起讨论LogicFlow的开发计划。

**Comment by wumail** [comment]
### 文档
- [ ] shouldUpdate相关内容补充 #1208 
- [ ] 控制面板选项介绍 #1127 
- [x] angular示例移植至示例站点 #972 @wumail 

### BPMN
- [ ] 转义问题 #1175 @wumail 
- [ ] 泳道 #312 
- [ ] check #571 
- [ ] #612 

### 优化
- [ ] 平板、手势操作 #471 #662   @sophia502
- [ ] 连线箭头复用 #1190 @wumail 
- [ ] bezier曲线终点箭头对齐 #951 @wumail 
- [ ] group节点缩放范围限制 #937 
- [ ] 插件参数透传 #892 
- [ ] 国际化功能移植 #569 
- [ ] transformModel优化 #901 
- [ ] util部分功能导出 #825 


### 功能
- [ ] 节点旋转 #390 
- [ ] 跳线 #1195 
- [ ] 中间件机制 #595 
- [ ] 边上插入节点后自动连接锚点 #1150 

@towersxu @wbccb @boyongjiong @sophia502

**Reply by wbccb** [reply]
很强！
